Formula/s/sonarqube.rb

```diff
@@ -49,6 +49,7 @@ def post_install
 
   def caveats
     <<~EOS
+      Conf: #{libexec}/conf/sonar.properties
```

Comment:
Are users meant to be able to customise this file? If so, we should move this elsewhere, because the contents of `libexec` will get blown away by `brew upgrade`.